test/org/sagebionetworks/bridge/validators/StudyParticipantValidatorTest.java

```diff
@@ -39,6 +42,12 @@
     @Mock
     private ExternalIdService externalIdService;
     
+    @Mock
+    private SubstudyService substudyService;
+    
+    @Mock
+    private Substudy substudy;
```

Comment:
Minor: Does substudy need to be mocked? Would it be easier just to create a new one for every test?

Reply:
Removed mock annotation, instantiated in @Before now